packages/app-check/README.md

```diff
@@ -48,8 +48,17 @@ A working example can be found here: [robingenz/capacitor-firebase-plugin-demo](
 
 ## Usage
 
+
 ```typescript
+// >= v7.1.0
```

Comment:
We want to keep it simple. Please revert your changes in the `Usage` section and just update the `initialize` method to use the `provider` property.